### PR TITLE
[Admin] (quick starts) Update labels and sequence of tabbed instructions in the Excel, Word, and PowerPoint quick starts.

### PR DESCRIPTION
**Pull request by kbrandl**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3bc2ee6](https://github.com/OfficeDev/office-js-docs-pr/pull/1181/commits/3bc2ee6013d3c6eab6c2c65e231d16d23f4795ad)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-choose-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/includes/quickstart-choose-editor.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/quickstarts/excel-quickstart-jquery.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172114249712-1181%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=70g9vM2dtwqrzDHJsbSeezjdO4DXkHpbb0lJBDcJRKQ%3D&st=2019-07-17T21%3A10%3A27Z&se=2019-08-17T21%3A15%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/quickstart-choose-editor.md [hunk 1]
```diff
@@ -1 +1 @@
-You can create an Office Add-in by using Visual Studio or any other editor. Tell us what editor you'd like to use by choosing one of the following tabs:
+You can create an Office Add-in by using the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office) or Visual Studio. The Yeoman generator creates a Node.js project that can be managed with Visual Studio Code or any other editor, whereas Visual Studio creates a Visual Studio solution. Tell us which one you'd like to use by choosing one of the following tabs.
```
Comment (AlexJerabek) [inline docs/includes/quickstart-choose-editor.md]: I'm not sure how I feel about "tell us". It sounds like we're collecting data or requiring an active input. Maybe some more passive like "Select your preferred Add-in creation method by choosing one of the following tabs."

File: docs/quickstarts/excel-quickstart-jquery.md [hunk 2]
```diff
@@ -14,7 +14,54 @@ In this article, you'll walk through the process of building an Excel task pane
...
+
+    ![Excel add-in button](../images/excel-quickstart-addin-3b.png)
+
+4. Select any range of cells in the worksheet.
+
+5. At the bottom of the task pane, choose the **Run** link to set the color of the selected range to yellow.
+
+    ![Excel add-in](../images/excel-quickstart-addin-3c.png)
+
+# [Visual Studio](#tab/visualstudio)
```
Comment (AlexJerabek) [inline docs/quickstarts/excel-quickstart-jquery.md]: Should this tab be called "Visual Studio Project"? To me, that speaks more to the creation method, which is on par with the "Yeoman Generator" tab name.

Comment (kbrandl) [reply docs/quickstarts/excel-quickstart-jquery.md]: Agreed that tab labels should indicate creation method -- but the creation method (i.e., what you're using to create the add-in) is either "Visual Studio" (which results in a Visual Studio solution) or "Yeoman Generator" (which results in a Node.js project).

File: docs/includes/quickstart-choose-editor.md (hunk 1 above)
Comment (kbrandl) [reply docs/includes/quickstart-choose-editor.md]: Agreed -- I never have liked that verbiage either. I've updated the sentence to this instead:  

Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[feb0d76](https://github.com/OfficeDev/office-js-docs-pr/pull/1181/commits/feb0d762f47aad1d8c15e11bef664662a7a35ae2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-choose-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/includes/quickstart-choose-editor.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/quickstarts/excel-quickstart-jquery.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-qs-tab-changes/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172143558507-1181%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Xymauy93RyyGbVf6Oi7MljzTdmKEFRLNPtLjISdc9n0%3D&st=2019-07-17T21%3A39%3A54Z&se=2019-08-17T21%3A44%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@AlexJerabek thanks for the feedback! I've responded here in this PR and made a change to the intro verbiage, per your recommendation. Let me know what you think; feel free to swing by my office if you'd like to chat about any of this in-person. Thanks!

**Pull request merged**